.github/ISSUE_TEMPLATE/roadmap-item.md

```diff
@@ -0,0 +1,31 @@
+---
+name: 'Roadmap Item'
+about: Create roadmap item for this project
+title: Roadmap Item =description=
+labels: kind/roadmap
```

Comment:
We probably need a 'needs-triage' label so if roadmap items are created they are easy to find for review. However volume of items should be low and we can use project board to track status of such items.

Reply:
Yes! I add the needs-triage label in [this PR](https://github.com/o3de/sig-simulation/pull/37). Will update this PR with the new changes by this week

Reply:
@lmbr-pip Updated the PR. Please help review!